HttpServerInNodeJS/index.js

```
@@ -0,0 +1,66 @@
+
+const express = require('express');
+const bodyParser = require('body-parser');
+const fs = require('fs');
+const dataPath = './tasks.json';
+
+const PORT = 8000;
+
+let app = express();
+app.use(bodyParser.json());
+
+// Read tasks list
+app.get('/tasks', (req, res) => {
+    console.log('Got GET request to /tasks')
+    fs.readFile(dataPath, (err, data) => {
+        if (err) throw err;
```

It is better to return a clear error to the user so he will know there was a problem along the way